### PR TITLE
Print project name

### PR DESCRIPTION
**Pull request by onon1101**
This pull request add a feature that print the project name "Sylinteri" in console.

```txt
➜  Sylinteri git:(printprojectname) ✗ ./build/SYLINTERI
   ______              __     _               _                       _
 .' ____ \            [  |   (_)             / |_                    (_)
 | (___ \_|   _   __   | |   __    _ .--.   `| |-'  .---.   _ .--.   __
  _.____`.   [ \ [  ]  | |  [  |  [ `.-. |   | |   / /__\ [ `/'`\]  [  |
 | \____) |   \ '/ /   | |   | |   | | | |   | |,  | \__.,  | |      | |
  \______.' [\_:  /   [___] [___] [___||__]  \__/   '.__.' [___]    [___]
             \__.'
[2024-10-29 21:33:30.829] [error] Need report file. Exit.

➜  Sylinteri git:(printprojectname) ✗
```

### CODE REVIEW

File: src/ProjectName.h [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+{
+    auto print_project_name() -> void {
+        printf("   ______              __     _               _                       _\n");
+        printf(" .' ____ \\            [  |   (_)             / |_                    (_)\n");
+        printf(" | (___ \\_|   _   __   | |   __    _ .--.   `| |-'  .---.   _ .--.   __\n");
+        printf("  _.____`.   [ \\ [  ]  | |  [  |  [ `.-. |   | |   / /__\\ [ `/'`\\]  [  |\n");
+        printf(" | \\____) |   \\ '/ /   | |   | |   | | | |   | |,  | \\__.,  | |      | |\n");
+        printf("  \\______.' [\\_:  /   [___] [___] [___||__]  \\__/   '.__.' [___]    [___]\n");
+        printf("             \\__.'\n");
+    }
```
Comment (c1ydehhx) [inline src/ProjectName.h]: Yo man this is crazy.

File: CMakeLists.txt [hunk 2]
```diff
@@ -7,7 +7,7 @@ set(CMAKE_CXX_FLAGS_DEBUG "-Werror -Og")
 
 include(FetchContent)
 
-find_package(CURL 7.81.0 EXACT REQUIRED)
+find_package(CURL REQUIRED)
```
Comment (c1ydehhx) [inline CMakeLists.txt]: It is not the part of this PR. Please revert it and re-create a PR for curl version issue.

File: src/ProjectName.h (hunk 1 above)
Comment (onon1101) [reply src/ProjectName.h]: yep, I know.
btw, I got the idea by the project https://github.com/ImCoriander/ZeroEye .

Comment (c1ydehhx) [reply src/ProjectName.h]: I think we can use this instead of hardcoded it into code. 
https://github.com/codewithnick/ascii-art

Comment (onon1101) [reply src/ProjectName.h]: emm....

But it's without modification if use hardcode. and I saw many repo write it like this.

Comment (onon1101) [reply src/ProjectName.h]: I mean the project name will not be modified in the future

Comment (onon1101) [reply src/ProjectName.h]: so hardcode in here is fine, right?

Comment (c1ydehhx) [reply src/ProjectName.h]: I think so. I'll consider that it's a good choice or not.

Comment (onon1101) [reply src/ProjectName.h]: ok. have a nice day.

File: src/ProjectName.h
Comment (c1ydehhx) [inline src/ProjectName.h:1]: Dispite the implementation is good or not, the file name of header should be lower case.

I suggest that `src/project_name.h`

Comment (onon1101) [reply src/ProjectName.h:1]: i got it

**Pull request merged**